nyc_taxi/dashboard.ipynb

```diff
@@ -11,12 +11,12 @@
+    "<img align='right' style='padding:0px 0px 0px 0px;' \n",
+    "src='./thumbnails/index.png'/>\n",
     "\n",
     "The [NYC Taxi trips](https://anaconda.org/jbednar/nyc_taxi/notebook) dataset is a well-studied data science example. Here we show how to build a simple dashboard for exploring 10 million taxi trips in a Jupyter notebook using [Datashader](http://datashader.org), then deploying it as a standalone dashboard using [Panel](http://panel.pyviz.org).\n",
     "\n",
-    "Running the dashboard requires having a live Python process running (not just a static webpage or anaconda.org notebook viewer).  Before you run the notebook or server, you'll need to set up a [conda](http://conda.pydata.org/miniconda.html) environment and run `conda install -c pyviz datashader panel holoviews`, and you'll need to [get the data file](https://anaconda.org/jbednar/nyc_taxi/notebook)."
+    "Running the dashboard requires having a live Python process running (not just a static webpage or anaconda.org notebook viewer).  Before you run the notebook or server, you'll need to set up a [conda](http://conda.pydata.org/miniconda.html) environment and run `conda install datashader panel holoviews`, and you'll need to [get the data file](https://anaconda.org/jbednar/nyc_taxi/notebook)."
```

Comment:
I'd suggest removing this paragraph:
- `requires having a live Python process running` -> This applies to all the notebooks, not just this one.
- setup (conda + data download) -> We assume users have set up the project with anaconda-project

Reply:
OK

Reply:
Please remove this, as per the second item in my previous comment:

> Before you run the notebook or server, you'll need to set up a [conda](http://conda.pydata.org/miniconda.html) environment and run `conda install datashader panel holoviews`, and you'll need to [get the data file](https://anaconda.org/jbednar/nyc_taxi/notebook)."